op-batcher/batcher/driver.go

```diff
@@ -274,11 +278,18 @@ func (l *BatchSubmitter) loop() {
 		case r := <-receiptsCh:
 			l.handleReceipt(r)
 		case <-l.shutdownCtx.Done():
+			// This removes any never-submitted pending channels, so these do not have to be drained with transactions.
+			// Any remaining unfinished channel is terminated, so its data gets submitted.
 			err := l.state.Close()
 			if err != nil {
-				l.Log.Error("error closing the channel manager", "err", err)
+				if errors.Is(err, ErrPendingAfterClose) {
+					l.Log.Warn("Closed channel manager, but pending channel(s) remain to be submitted")
```

Comment:
Should change this log message to mention that the pending channels will be submitted? It sounds like this is an invalid state and we've left the channel unclosed.

Reply:
Changed to "Closed channel manager on shutdown with pending channel(s) remaining - submitting", similarly for L2 reorg warning.